api/v1/openapi.yaml

```diff
@@ -1469,9 +1469,12 @@ definitions:
         description: Docker network ID
         type: string
       pod-name:
-        description: K8s pod for this endpoint
+        description: K8s pod for this endpoint(Deprecated, use K8sPodName and K8sNamespace instead)
         type: string
```

Comment:
I think open API can have `deprecated: true` in here

Reply:
I tried doing this, but we are using openapi spec version 2.x which does not support deprecated fields for properties. https://stackoverflow.com/questions/49870109/how-to-annotate-a-field-as-deprecated-in-openapi-swagger-2-0